### DISCUSSION TITLE
Follow users

### DISCUSSION THREAD

**Discussion by wesleybertipaglia** [dailydotdev/daily#1044] [Feature Requests]
Hi, is it possible to add an option to follow users? I have some users that I enjoy reading their articles, but I always have to search for them by username. I know it's a difficult feature to implement, but it would be incredible to be able to follow these users.

**Comment by nimrodkra** [comment]
Hi Wesley, thanks for the suggestion! Would you mind sharing with us
several examples of users you would want to follow on daily.dev and why
would you want to follow them? It will help our product team a better
understanding of the use case.

**Reply by wesleybertipaglia** [reply]
I'm sorry, I actually meant authors. For example, there is no way to connect with an author to always receive their new posts:
![image](https://github.com/dailydotdev/daily/assets/49965963/28be996f-e99e-41ea-8dd3-d0df319dd5d6)

**Comment by nimrodkra** [comment]
Got it, thanks for giving us more context. We are exploring an idea that
should solve it but the timeline to build it is still unclear. We’ll keep
you posted

**Reply by wesleybertipaglia** [reply]
Nice @nimrodkra. I'm here to help if you need anything 🙂
